client/system/info_linux.go

```diff
@@ -122,3 +124,51 @@ func sysInfo() (serialNumber string, productName string, manufacturer string) {
 	si.GetSysInfo()
 	return si.Product.Version, si.Product.Name, si.Product.Vendor
 }
+
+// getRunningProcesses returns a list of running process paths.
+func getRunningProcesses() ([]string, error) {
```

Comment:
Duplicated codes between darwin, windows, Linux. Instead of copy+past, you could move this code into a new file and in the build tags you can define the list of supported operation systems.